x/metadata/keeper/msg_server.go

```diff
@@ -368,6 +368,83 @@ func (k msgServer) DeleteContractSpecification(
 	return types.NewMsgDeleteContractSpecificationResponse(), nil
 }
 
+func (k msgServer) AddContractSpecToScopeSpec(
+	goCtx context.Context,
+	msg *types.MsgAddContractSpecToScopeSpecRequest,
+) (*types.MsgAddContractSpecToScopeSpecResponse, error) {
+	defer telemetry.MeasureSince(time.Now(), types.ModuleName, "tx", "AddContractSpecToScopeSpec")
+	ctx := sdk.UnwrapSDKContext(goCtx)
+	contractSpec, found := k.GetContractSpecification(ctx, msg.ContractSpecificationId)
+	if !found {
+		return nil, fmt.Errorf("contract specification not found with id %s", msg.ContractSpecificationId)
+	}
+	if err := k.ValidateAllOwnersAreSigners(contractSpec.OwnerAddresses, msg.Signers); err != nil {
```

Comment:
the object being updated is the scope specification ... not the contract specification.  The contract specification owners are not required to sign as their object is not changing.